integration/spark/src/test/scala/org/apache/spark/carbondata/restructure/AlterTableRevertTestCase.scala

```diff
@@ -30,6 +32,13 @@ import org.apache.carbondata.spark.exception.ProcessMetaDataException
 class AlterTableRevertTestCase extends QueryTest with BeforeAndAfterAll {
 
   override def beforeAll() {
+    new MockUp[MockClassForAlterRevertTests]() {
```

Comment:
can we mock up the static method directly?

Reply:
i checked for static methods, but all were defined in scala object not class, so i thought i can do this way.

Reply:
for scala, The class is CarbonSessionCatalogUtil$.  The method name is updateCatalogTableForAlter.  It is private, but not a static method.

Reply:
yeah, i tried that, but since im using MockUp[T], it needs actual scala class name, not the object, as MockUp identifies class, so i followed this similarly by referring the `TableStatusBackupTest`

Reply:
@QiangCai Since CarbonSessionCatalogUtil is a singleton object in scala, we cant mock its method directly, so i think this should be fine right for now?

Reply:
ok